src/lib.ts

```diff
@@ -92,9 +96,64 @@ export function advCost(whichTest: number): number {
   }
 }
 
+function statTurnsSaved(whichTest: number): number {
+  if (whichTest === CommunityServiceTests.HPTEST)
+    return Math.floor((myMaxhp() - myBuffedstat($stat`Muscle`) - 3) / 30);
+
+  let stat: Stat;
+  switch (whichTest) {
+    case CommunityServiceTests.MUSTEST:
+      stat = $stat`Muscle`;
+      break;
+    case CommunityServiceTests.MYSTTEST:
+      stat = $stat`Mysticality`;
+      break;
+    case CommunityServiceTests.MOXTEST:
+      stat = $stat`Moxie`;
+      break;
+    default:
+      return 0;
+  }
+  return Math.floor((myBuffedstat(stat) - myBasestat(stat)) / 30);
+}
+
+function logRelevantStats(whichTest: number): void {
```

Comment:
This can be cleaner. There's a lot of repetitive code here. I can clean this up later